tests/Installer/PackagesInstallerTest.php

```diff
@@ -56,13 +56,13 @@ public function testCanInstallWithMocks(): void
      */
     public function testCanMergeDefaultPackagesWhenInstalling(): void
     {
-        $composer     = $this->createMock(Composer::class);
-        $typeResolver = $this->createMock(ProjectTypeResolver::class);
-        $depInstaller = $this->createMock(DependencyInstaller::class);
-        $io           = $this->createMock(IOInterface::class);
+        $composerMock = $this->createMock(Composer::class);
+        $typeResolverMock = $this->createMock(ProjectTypeResolver::class);
+        $depInstallerMock = $this->createMock(DependencyInstaller::class);
+        $ioMock = $this->createMock(IOInterface::class);
 
         // Simulate magento 1 project
-        $typeResolver
+        $typeResolverMock
             ->method('resolve')
             ->willReturn('magento1');
```

Comment:
note to self: would expect these tests to fail

Reply:
Test was just for the merging strategy and the mapping was part of the test case. However, the implementation still had m1 references even though support is dropped so that mapping was also cleaned up and the test case updated to use a fictional package and type instead